src/main/java/seedu/duke/storage/PatientList.java

```diff
@@ -0,0 +1,36 @@
+package seedu.duke.storage;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import seedu.duke.record.Patient;
+
+public class PatientList {
+    private List<Patient> patientList;
```

Comment:
Perhaps making this to static would be better so that this member will be shared by all instances of the object PatientList.